package.json

```diff
@@ -147,7 +147,7 @@
   },
   "dependencies": {
     "@accessible/visually-hidden": "^2.0.0",
-    "@react-hook/switch": "^1.3.0",
+    "@react-hook/switch": "^1.3.1",
```

Comment:
I fixed one other thing that was not working perfectly in there. Can you bump this to `1.3.3` really quick :)